tests/storage/iso/test_cifs_iso_sr.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import logging
+import os
+from conftest import copy_tools_iso_to_iso_sr, check_iso_mount_and_read_from_vm
+from conftest import remove_iso_from_sr, cifs_iso_device_config
+
+# Requirements:
+# - one XCP-ng host with a SAMBA/CIFS share for the SR and a VM
```

Comment:
Please use one line per requirement, and tell where the data comes from. This comment applies to all test files.

Some of our tests have better requirements that others and the ones you took as example were not the best.

Here's a "good" example:

```
# Requirements:
# From --hosts parameter:
# - host(A1): first XCP-ng host >= 8.2 (+ XAPI patch allowing to select migration network for evacuate)
# - hostA2: Second member of the pool.
# From --vm parameter
# - A VM to import
# From data.py or --sr-device-config parameter: configuration to create a new NFS SR.
# From --second-network parameter (for most tests)
# - second_network: A 2nd physical network of the pool with PIF plugged and configured with an IP on all hosts
#   Must NOT be the management interface.
#   Must NOT be the network used to access the NFS SR.
#   This network will be disconnected at some point during the tests
```